packages/shared/lib/core/network/enums/destination-network.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum DestinationNetwork {
+    Shimmer = 'Shimmer',
+    ShimmerEvm = 'Shimmer EVM',
```

Comment:
I suppose this doesn't need to adhere to the normal `enum` conventions since the text doesn't need to be localized?